api-response.go

```diff
@@ -17,16 +17,198 @@
 package main
 
 import (
+	"encoding/xml"
 	"net/http"
 
 	"github.com/minio/minio/pkg/fs"
 )
 
-// Reply date format
 const (
+	// Reply date format
 	rfcFormat = "2006-01-02T15:04:05.000Z"
```

Comment:
No AB it is not fully rfc3339. S3 date doesn't have nanosecond element. It's a shorter version of rfc3339 perhaps.